src/plugins/data/common/search/aggs/buckets/range_key.ts

```diff
@@ -19,20 +19,41 @@
 
 const id = Symbol('id');
 
+type Ranges = Array<{
+  from: string | number | undefined;
+  to: string | number | undefined;
+  label?: string;
+}>;
```

Comment:
nit: because all props here can be omitted, would it make sense to declare it as `Partial<{...}>`?